### ISSUE TITLE
example usage for --umbrella projects`

### ISSUE THREAD

**Issue by ellismarkf** [phoenixframework/tailwind#9]
The current installation instructions assume that users are not working in or starting a new umbrella project, ie `mix new my_app --umbrella`.

I ran into a lot of issues trying to get this to work in a new umbrella project, and eventually just gave up and started the project fresh as a default `mix` project.

I'm pretty new to Elixir, so it could just be that I'm missing something pretty obvious, but I thought that this would Just Work™ and was surprised when it didn't. 

Would it be possible to add example usage for `--umbrella` projects?

**Comment by a-blksheep** [comment]
I second this :) Attempted to follow along with the instructions found [here](https://fly.io/phoenix-files/tailwind-standalone/) but encountered the following issues:

1. Tailwind config file is created as: ./assets/tailwind.config.js instead of ./apps/my_umbrella_app_web/assets/tailwind.config.js. Admittedly, while I did manage to tinker with the config a little to get it sort of working:

```
config :tailwind,
  version: "3.0.7",
  default: [
    args: ~w(
      --config=tailwind.config.js
      --input=css/app.css
      --output=../priv/static/assets/app.css
    ),
    cd: Path.expand("../apps/my_umbrella_app_web/assets", __DIR__)
  ]
```
Admittedly this still created the config file in the ./assets dir. I then manually moved the generated file into the correct location.

2. Once I got my config in the correct place, Phoenix spat back errors regarding not being able to find phoenix.css. Admittedly this may be less of an issue since I will inevitably end up removing the relevant file eventually.

Unfortunately, I can't share the configuration I had as I'd already reverted my changes :( However, I'm inclined to spin up a quick dummy project just for kicks and giggles and to see if that helps you guys a bit.

PS: Thanks to all the Phoenix guys for what is an absolutely incredible framework :) It (and Elixir by extension) are truly a breath of fresh air.

**Comment by a-blksheep** [comment]
Perhaps, I'm just an idiot? I'm also guessing that the missing phoenix.css file is probably just a mis-configuration on my part?!? But, here's the output I get:

```
[debug] ** (Phoenix.Router.NoRouteError) no route found for GET /assets/phoenix.css (TailwindDemoWeb.Router)
    (phx_tailwind_demo_web 0.1.0) lib/phoenix/router.ex:406: TailwindDemoWeb.Router.call/2
    (phx_tailwind_demo_web 0.1.0) lib/phx_tailwind_demo_web/endpoint.ex:1: TailwindDemoWeb.Endpoint.plug_builder_call/2
    (phx_tailwind_demo_web 0.1.0) lib/plug/debugger.ex:136: TailwindDemoWeb.Endpoint."call (overridable 3)"/2
    (phx_tailwind_demo_web 0.1.0) lib/phx_tailwind_demo_web/endpoint.ex:1: TailwindDemoWeb.Endpoint.call/2
    (phoenix 1.6.5) lib/phoenix/endpoint/cowboy2_handler.ex:54: Phoenix.Endpoint.Cowboy2Handler.init/4
    (cowboy 2.9.0) /home/blksheep/Code/tinkering/phx_tailwind_demo_umbrella/deps/cowboy/src/cowboy_handler.erl:37: :cowboy_handler.execute/2
    (cowboy 2.9.0) /home/blksheep/Code/tinkering/phx_tailwind_demo_umbrella/deps/cowboy/src/cowboy_stream_h.erl:306: :cowboy_stream_h.execute/3
    (cowboy 2.9.0) /home/blksheep/Code/tinkering/phx_tailwind_demo_umbrella/deps/cowboy/src/cowboy_stream_h.erl:295: :cowboy_stream_h.request_process/3
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3

```

That said, I created two quick demos based on @chrismccord's [blog post](https://fly.io/phoenix-files/tailwind-standalone/)

- https://github.com/a-blksheep/phx_tailwind_demo_umbrella
- https://github.com/a-blksheep/phx_tailwind_demo

Just in case anyone is interested.

Thanks once again.

**Comment by chrismccord** [comment]
Closed via #3
